src/charm.py

```diff
@@ -200,6 +201,10 @@ def on_pebble_ready(self, _: EventBase) -> None:
         self._container.add_layer(self._name, pebble_layer, combine=True)
         self._container.autostart()
 
+        if (version := self._agent_version) is not None:
+            self.unit.set_workload_version(version)
+        else:
+            logger.debug("Could not get agent version")
```

Comment:
This may look suspicious when looking at the debug log. Clarify it's "not a big deal"?

Reply:
Done.